apis/basic/composition.yaml

```diff
@@ -31,6 +31,11 @@ spec:
         - type: FromCompositeFieldPath
           fromFieldPath: spec.parameters.region
           toFieldPath: spec.forProvider.region
+    - name: network-id-tags
+      patches:
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.parameters.id
+          toFieldPath: spec.forProvider.tags[networks.aws.platform.upbound.io/network-id]
```

Comment:
nit: i would regroup and place this PatchSet next to network-id

Reply:
done